components/work-schedule/work-schedule.module.css

```diff
@@ -0,0 +1,21 @@
+.div {
+  margin-top: 1.75rem;
+  margin-bottom: 1.75rem;
```

Comment:
Add `border-radius: 10px`. Helps to have a rounded shape to be consistent with existing designs.